values.yaml

```diff
@@ -2,31 +2,81 @@
 image:
   name: traefik
   tag: 2.0.5
+
+#
+# Configure the deployment
 #
 deployment:
+  # Number of pods of the deployment
   replicas: 1
+
+rollingUpdate:
+  maxUnavailable: 1
+  maxSurge: 1
+
+# Additional arguments to be passed at the startup of traeffik
+additionalArguments: []
+#  - "--argument1"
```

Comment:
I'd gladly get a better idea of example ^-^

Reply:
In the spirit of #21, WDYT about 

```yml
#  - --provider.file.filename=/etc/traefik.dynamic.yml
```

?

Reply:
That would imply we provide a way to mount this as a configmap somewhere which I'm not doing